### PR TITLE
Update documentation by new fields

### PR DESCRIPTION
**Pull request by SergeyAleksejev**
There is new parameter added to file - system_name

### CODE REVIEW

**Review by pgorbachev — APPROVED** [review]

File: docs/configuration.adoc [hunk 1]
```diff
@@ -68,9 +69,19 @@ Components Registry is originally stored as configuration files with ConfigSlurp
...
+        parentComponent      = <component id> // grouping components in DMS
+        clientCode           = <client code> // if component is client specific
+        solution             = <true|false> // true - meta-component
+
+        componentOwner   = "username of component's owner" //without domain part. For example: salekseev
+        releaseManager   = "username of component's release manager" 
+        securityChampion = "username of component's security officer" 
 
+        buildSystem = <build tool> // MAVEN, GRADLE etc. (default for all versions)
+        system      = <{system_name}>
```
Comment (qu0d) [inline docs/configuration.adoc:83]: Please, comment system field

**Comment by octopusden** [comment]
@SergeyAleksejev , what are you plans regarding the pull request?

File: docs/configuration.adoc [hunk 2]
```diff
@@ -1,12 +1,13 @@
 :icons: font
 :header: Components Registry Configuration
+:system_name: Name of system
```
Comment (octopusden) [inline docs/configuration.adoc:3]: what is this for?

**Review by pgorbachev — DISMISSED** [review]

File: docs/configuration.adoc (hunk 1 above)
Comment (qu0d) [reply docs/configuration.adoc:82]: // used for differentiation between different corporate policies (license control, etc)

Comment (qu0d) [reply docs/configuration.adoc:82]: // used for devide corporate policies (license control, etc)

**Review by qu0d — APPROVED** [review]

**Pull request merged**